docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1368,12 +1395,17 @@ Following is the beautified response of sample polling location request after th
 
 For the Import Operation, the polled status includes an indication of `$import` and the location of the OperationOutcome NDJSON files and the corresponding failure and success counts.
 
+For S3 exports, the bulk data feature may use presigned urls.  To enable presigned urls, the authentication type must be `hmac` and `fhirServer/bulkdata/storageProviders/(source)/presigned` must be `true`.  The urls become:
+
+```
+https://s3.appdomain.cloud/fhir-example/Patient_1.ndjson?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=fcEXbf9cc1ac49e99eEX77%2F20210228%2Fus%2Fs3%2Faws4_request&X-Amz-Date=20210EXT160538Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-Amz-Signature=5ecfc207546b9737fd38d4f0EXEX1c58f4f82976338a44c
+```
```

Comment:
how long is the presigned URL valid for?  is it configurable?

Reply:
86400 seconds, it is not configurable. 
It could be... we'd just have to add a new configuration for it.

Reply:
I think we should
1. mention that here
2. open a new issue for making it configurable

I'll work on number 2.

Reply:
it is

Reply:
https://github.com/IBM/FHIR/issues/2053